### PR TITLE
fix(javascript): regression for `hasOwnProperty`

### LINKED ISSUE #4602: Regression: Unconditionally split `hasOwnProperty` call to multiple lines.
**Prettier 1.13.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEALAhmgNVwBsBXOTAXh3wHkB3KABQCcIAHOFmATwApcAGkwAjAJQBuEIJAcYAS3TJQuFm3pNVCNMhC4AbhHkATaSBEtcYANZwYAZXZX5UAObIYLcjJdouMVlxXAFtcZAAzEj8ZACs0AA8AIUsbO3tcYLgAGRc4CKi4GScWPxZdEVwRHmJoM3YWFxgAdRMYbGQADgAGIrY-Jst2XXq4Uv08mRY4AEdSeSnAkLCkSOJokD9g+Q8vQo2XV2I4AEVSCHh8tb2YSpbjNuQAJhlPXHliA4BhCGDQ3ShoBMQKQ-AAVSo6FYFAC+0KAA)
```sh
--parser babylon
```

**Input:**
```jsx
const hasValue = hasOwnProperty(a, b);
```

**Output:**
```jsx
const hasValue = hasOwnProperty(
  a,
  b
);

```

**Expected behavior:**
```jsx
const hasValue = hasOwnProperty(a, b);
```

### PR DESCRIPTION
**Pull request by ikatyang**
Fixes #4602

### CODE REVIEW

File: src/language-js/printer-estree.js [hunk 1]
```diff
@@ -3418,30 +3418,31 @@ function shouldGroupFirstArg(args) {
...
-  pipeK: true, // Ramda
-  compose: true, // Ramda, Redux
-  composeFlipped: true, // Not from any library, but common in Haskell, so supported
-  composeP: true, // Ramda
-  composeK: true, // Ramda
-  flow: true, // Lodash
-  flowRight: true, // Lodash
-  connect: true // Redux
-};
+const functionCompositionFunctionNames = new Set([
```
Comment (lydell) [inline src/language-js/printer-estree.js]: Hehe, I proposed using a Set in the original PR :)

https://github.com/prettier/prettier/pull/4431#discussion_r186300367

**Comment by IvanGoncharov** [comment]
@ikatyang Thanks for the super fast fix 🏁🏎
I'm just wondering when to expect next release with this fix included?

**Comment by ikatyang** [comment]
I'm not sure when but we should make these regression fixes available as soon as possible.

And it seems Jest also encountered this issue https://github.com/facebook/jest/pull/6344#issuecomment-393213287. 

@duailibe what do you think?

**Comment by duailibe** [comment]
@ikatyang makes sense. I'll release 1.13.3

**Pull request merged**